workspace.rkt

```diff
@@ -77,3 +81,11 @@
   (when (regexp-match (get-module-suffix-regexp) uri)
     (clear-old-queries/doc-close uri)
     (hash-remove! open-docs (string->symbol uri))))
+
+(define (didChangeConfiguration params)
+  (match-define (hash-table ['settings settings]) params)
+
+  (define key '(resyntax enable))
+  (when (jsexpr-has-key? settings key)
+    (set-resyntax-enabled! (jsexpr-ref settings key #t))))
```

Comment:
I think `jsexpr-ref` only takes two arguments? They are `jsexpr` and `keys` (access chain), and no fallback value.

Reply:
Yes, I forgot to remove that `#t`.